### ISSUE TITLE
How to tune the parameters of detection algorithms?

### ISSUE THREAD

**Issue by JR-Jared** [HSE-LAMBDA/roerich#8]
I use roerich on my own dataset, but the result is not so good. 

![figure](https://user-images.githubusercontent.com/55573326/194822191-e935fce0-57f3-4930-90b3-2246daf2abd6.png)

There are comments in the class `ChangePointDetection`: 

        Parameters
        ----------
        scaler: A scaler object is used to scale an input data. The default one is `SmaScalerCache`
        metric: A loss function during optimize step of NN. Can be one of the following KL_sym, KL, JSD, PE, PE_sym, Wasserstein
        window_size: A size of a window when splitting input data into train and test arrays
        periods: A number of previous data-points used when constructing autoregressive matrix
        lag_size: A distance between train- and test- windows
        step: Each `step`-th data-point is used when creating the input dataset
        n_epochs: A number of epochs during training NN
        lr: A learning rate at each step of optimizer
        lam: A regularization rate
        optimizer: One of Adam, SGD, RMSprop or ASGD optimizers
        debug: default zero

How to tune the parameters of the change point algorithms, like `periods`, `window_size` , `lag_size` or `step`?

**Comment by hushchyn-mikhail** [comment]
@JR-Jared, hi!

The key parameters here are window_size and lag_size. The general recommendation is to choose values such that only one change point is within 2 * (window_size + lag_size).
